cli/src/test/kotlin/com/amazon/ionschema/cli/commands/repair/FixTransitiveImportsCommandTest.kt

```diff
@@ -0,0 +1,106 @@
+        private val baseOutputDir = File("build/tmp/test/").canonicalPath
+        val timestamp = Instant.now().truncatedTo(ChronoUnit.SECONDS)
+
+        private fun runCommand(strategy: ImportStrategy) {
+            val outputDir = File("$baseOutputDir/FixTransitiveImportsCommand/$strategy-test-output-$timestamp")
+            IonSchemaCli().parse(
+                arrayOf(
+                    "repair", "fix-transitive-imports",
+                    "-s", "$strategy",
+                    "-d", "$outputDir",
+                    "$testResourceDir/FixTransitiveImportsCommand/input/",
+                )
+            )
+        }
+
+        fun testForStrategy(strategy: ImportStrategy): Stream<DynamicNode> {
+
+            val outputDir = File("$baseOutputDir/FixTransitiveImportsCommand/$strategy-test-output-$timestamp")
+            val expectationDir = File("$testResourceDir/FixTransitiveImportsCommand/output-$strategy")
+
+            val testNodes = mutableListOf<DynamicNode>()
+            expectationDir.walk()
+                .filter { it.isFile }
+                .forEach { expectedFile ->
+                    val relativeFile = expectedFile.relativeTo(expectationDir)
+                    val actualFile = outputDir.resolve(relativeFile)
+
+                    // This should not be brittle, and should probably not require the expected outputs to be updated.
+                    testNodes.add(
+                        dynamicTest("Check $relativeFile are Ion equivalent", actualFile.toURI()) {
+                            Assertions.assertEquals(
+                                ION.loader.load(expectedFile),
+                                ION.loader.load(actualFile),
+                            )
+                        }
+                    )
+                    // This test could be brittle, but we need it to ensure (a) that we don't wipe out any comments in the
+                    // ISL, and (b) that we don't suddenly start making large style-related changes to the original files.
+                    // For some changes this test may fail, and sometimes we might want to update the expected outputs
+                    // (e.g. if there's some intended whitespace change).
+                    // To do so, run:
+                    // ./ion-schema-cli repair fix-transitive-imports -s <STRATEGY> \
+                    //     cli/src/test/resources/FixTransitiveImportsCommand/input/ \
+                    //     -d cli/src/test/resources/FixTransitiveImportsCommand/output-<STRATEGY>/
+                    testNodes.add(
+                        dynamicTest("Check $relativeFile are text equivalent", actualFile.toURI()) {
+                            Assertions.assertEquals(
+                                expectedFile.readText(),
+                                actualFile.readText(),
+                            )
+                        }
+                    )
```

Comment:
How are failures here presented to the user? I think it would be great if they failed separately from the tests that are expected not to be brittle. Can we group them together into something named obviously, like "PossiblyBrittleTests"?

Reply:
Failures here are only presented to someone developing the tool. Not to a user of the tool. The test cases are named, and the potentially brittle ones are called `"Check $relativeFile are text equivalent"`. There is no risk of them failing unless you are messing with the source code of the tool and changing the text output.

Maybe brittle isn't the right word for this.

Reply:
Yeah I understand that. Still, as a developer, I find it nice to have tests that may fail in the future due to external factors marked explicitly. Could they start failing if IonJava's text writer changed its whitespacing strategy, for example?

Reply:
Added a "⚠ Fragile ⚠" prefix to the name of the test cases.